asciidoctorj-api/src/main/java/org/asciidoctor/ast/Catalog.java

```diff
@@ -0,0 +1,18 @@
+package org.asciidoctor.ast;
+
+import java.util.List;
+
+/**
+ * The asciidoctor catalog contains data collected by asciidoctor that is useful to a converter.
+ * The catalog is considered an internal structure by asciidoctor, and subject to change.
```

Comment:
Not sure we should state the second line. Maybe causes consufions in API users.

Reply:
Maybe you are right here. I got this intel from the issue discussion and found it valuable. Happy to adjust as needed.

Reply:
You know what? I think you are right. If asciidoctor docs/code stated this somewhere I could repeat it, but I've not yet found it stating this.  I'll turf the 2nd line.

Reply:
Or as @robertpanzer suggested, maybe it makes more sense in the getter.
It just seems to me the class description is not the place for internal details, but what do I know :man_shrugging:

Reply:
That's indeed a difficult question and I am probably wrong most of the time😂

What I find good is to mention that this is subject to change if it's also not directly part of the public API of asciidoctor.

Reply:
Oh I kind of missed that getter point.  Ya, makes sense, I think.  
I'll move it but also get rid of the 2nd line.

Reply:
Ok @robertpanzer, I'll use that wording then.